### PR TITLE
Stylelint update and documentation update

### PR DESCRIPTION
**Pull request by mdvanes**
Updated stylelint to 7.7.1 and updated the readme to have an example grunt config.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +23,23 @@ In your Gruntfile, add the line:
 grunt.loadNpmTasks( 'grunt-stylelint' );
 </pre>
 
-Running and configuring
---------------------
+## Stylelint task
```
Comment (ntwb) [inline README.md]: Lowercase s, stylelint is spelt all lowercase, please 😄

File: README.md [hunk 2]
```diff
@@ -24,9 +23,23 @@ In your Gruntfile, add the line:
...
 
 _Run this task with the `grunt stylelint` command._
 
 You can specify the targets and options for the task using the normal Grunt configuration – see Grunt's [guide on how to configure tasks](http://gruntjs.com/configuring-tasks) in general.
+
+For more explanations of the lint errors Stylelint will throw at you please visit http://stylelint.io/.
```
Comment (ntwb) [inline README.md]: Another instance, stylelint please

**Comment by mdvanes** [comment]
Updated spelling.

File: README.md [hunk 3]
```diff
@@ -24,9 +23,23 @@ In your Gruntfile, add the line:
...
+
+### Usage examples
+
+In this example, running `grunt stylelint:all` (or `grunt stylelint` because `stylelint` is a [multi task](http://gruntjs.com/configuring-tasks#task-configuration-and-targets)) will lint the project's CSS and SASS files in the `css` and `sass` directories and their subdirectories, using the default stylelint options or the options specified in the `.stylelintrc` in the root of the project.
```
Comment (Volker-E) [inline README.md]: And “SASS” is written “Sass”.

**Pull request merged**